assets/js/admin/lesson-quick-edit.js

```diff
@@ -11,7 +11,7 @@
 
 		// get the post ID
 		var postId = 0;
-		if ( typeof( id ) == 'object' ) {
+		if ( !Number.isInteger( id ) ) {
```

Comment:
Looks like the value of `id` is actually a button - `<button type="button" class="button-link editinline" aria-label="Quick edit “Lesson 1” inline" aria-expanded="true">Quick&nbsp;Edit</button>`. 

I wonder if the original intention of this check was supposed to be that `this` actually had a `getId` function before calling it.

Reply:
It's calling the `getId` of Wordpress's inlineEditPost, which expects a DOM node and looks for its parent row's id attribute. Switched this to an `instanceof` check that makes more sense.